src/components/Inventory/view/QuantitiesMenu.tsx

```diff
@@ -71,7 +71,7 @@ export default function QuantitiesMenu(props: QuantitiesMenuProps): JSX.Element
           icon='menuVertical'
           type='passive'
           priority='ghost'
-          size='small'
+          size='medium'
```

Comment:
This makes the row size grow quite a bit, which prompted the change in https://github.com/terraware/web-components/pull/306 to match the table spec.